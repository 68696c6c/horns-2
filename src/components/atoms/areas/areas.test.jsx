/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Areas from '.'

describe('Areas', () => {
  it('should render as default', () => {
    const { container } = render(
      <Areas>
        <div>a</div>
        <div>b</div>
      </Areas>
    )
    expect(container).toMatchSnapshot()
  })
})
