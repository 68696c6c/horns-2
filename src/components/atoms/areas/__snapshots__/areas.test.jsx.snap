// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Areas should render as default 1`] = `
.emotion-0 {
  display: block;
}

@media (min-width:480px) {
  .emotion-0 {
    display: grid;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    padding-left: calc(((100vw - 1200px) / 2));
    padding-right: calc(((100vw - 1200px) / 2));
  }
}

<div>
  <div
    class="emotion-0"
  >
    <div>
      a
    </div>
    <div>
      b
    </div>
  </div>
</div>
`;
